### PR TITLE
Implement search location by city name

### PR DESCRIPTION
**Pull request by ndedic**

### CODE REVIEW

File: Location_test.go [hunk 1]
```diff
@@ -21,8 +21,17 @@ func TestGetLocations(t *testing.T) {
 }
 
 func TestSearchForLocation(t *testing.T) {
-	goodLocation := "11238"
-	badLocation := "11"
+	postcode := "11238"
```
Comment (adamo57) [inline Location_test.go]: is there a way that we can test for failure in these tests?

**Comment by adamo57** [comment]
@ndedic you're going to have to update the Location file. There was a big change that needed to happen and just so happened to get finished and merged in last night. As soon as it is updated, then I will merge and close 👍

**Comment by ndedic** [comment]
Ok, nice change btw. This should be done now.

**Comment by adamo57** [comment]
thank you so much for the PR! :D

**Pull request merged**